modules/improvedigitalBidAdapter.js

```diff
@@ -10,6 +10,70 @@ const BIDDER_CODE = 'improvedigital';
 const RENDERER_URL = 'https://acdn.adnxs.com/video/outstream/ANOutstreamVideo.js';
 const VIDEO_TARGETING = ['skip', 'skipmin', 'skipafter'];
 
+class Razr {
```

Comment:
Why creating a class with all `static` functions?  
You could just define and use functions without defining a class.

Reply:
It was to group related methods and variables in a single entity. It looks nicer than for example prefixing the function and variable names with the "razr". Another options would be to replace the static class with a plain JS object. What is a preferred convention?

Reply:
I don't think there is a preferred convention. You can leave like this. I just prefer clean functions over OOP (when OOP is not necessary).

Reply:
I agree, using a static class is a bit of a stretch here. However, we have a brand new Improve Digital adapter that will replace the existing one hopefully in the next couple of weeks. I'll have the class replaced by an object literal similar to [here](https://github.com/prebid/Prebid.js/blob/master/modules/bluebillywigBidAdapter.js#L31) so if that's ok, please review as is.

Reply:
@FilipStamenkovic can we proceed?

Reply:
@ChrisHuie can you assign someone else to take a look at this PR? At the moment I don't have a spare time to dig into it.

Reply:
Hey @ChrisHuie. Would it possible to get a rough ETA for when the PR will get reviewed? Thanks

Reply:
@jbartek25 added a reviewer to take a look :)

Reply:
I don't necessarily feel qualified to make the call on this question. From my perspective, it works and you're planning to replace this in the future, so I'm failing to see any consequence really.